src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/DialogActivities/Queries/Get/GetDialogActivityDto.cs

```diff
@@ -0,0 +1,21 @@
+using Digdir.Domain.Dialogporten.Application.Features.V1.Common.Localizations;
+using Digdir.Domain.Dialogporten.Domain.Dialogs.Entities.Activities;
+
+namespace Digdir.Domain.Dialogporten.Application.Features.V1.ServiceOwner.DialogActivities.Queries.Get;
+
+public sealed class GetDialogActivityDto
+{
+    public Guid Id { get; set; }
+    public DateTimeOffset? CreatedAt { get; set; }
+    public Uri? ExtendedType { get; set; }
+    
+    public DialogActivityType.Enum Type { get; set; }
+    
+    public DateTimeOffset? DialogDeletedAt { get; set; }
```

Comment:
Tenker at dette feltet bare kan hete deletedAt. Dersom dialogen slettes, så slettes hele aggregatet. Samme for list. Og det er helt greit at det ligger på hver element i liste apiet tenker jeg 😊